.circleci/common.sh

```diff
@@ -58,11 +58,11 @@ function install_deps_pytorch_xla() {
   pip install hypothesis
   pip install cloud-tpu-client
   pip install absl-py
-  pip install --upgrade numpy>=1.18.5
+  pip install --upgrade "numpy>=1.18.5"
   pip install --upgrade numba
 
   # Using the Ninja generator requires CMake version 3.13 or greater
-  pip install cmake>=3.13 --upgrade
+  pip install "cmake>=3.13" --upgrade
```

Comment:
any reason for these changes?

Reply:
I accidentally found this bug while debugging ci container. `pip install cmake>=3.13` is equivalent to `pip install cmake > "=3.13"` and creates a file named `=3.13`.